### ISSUE TITLE
[Bug]: Formatting SQL statements, there is a space after ‘<=>’ formatting

### ISSUE THREAD

**Issue by Huangxiao-mas** [oceanbase/odc#3115]
### ODC version

ODC431

### OB version

independent

### What happened?

Formatting SQL statements, there is a space after ‘<=>’ formatting
![image](https://github.com/user-attachments/assets/eee84a40-b4de-471c-9f09-2ab6477f7107)
![image](https://github.com/user-attachments/assets/580fa88a-c4b0-4f1d-9e0e-38396ab473b6)

### What did you expect to happen?

。。

### How can we reproduce it (as minimally and precisely as possible)?

```
SELECT pk FROM table1_hash_to_days_pk_to_days_pk_subpart_by_range_columns_pk_su WHERE ( ( col_bigint <=> 5134 AND col_char_20 IS NULL ) AND ( col_bigint > 5134 OR (  ( col_timestamp_6 < DATE_ADD( DATE_ADD( DATE_ADD( CONCAT( CAST( 2015 + 1 AS CHAR(20) ) , '-06-15 01:00:00.000003') , INTERVAL '01 05:05:05' DAY_SECOND ) , INTERVAL '05:05.000005' MINUTE_MICROSECOND ) , INTERVAL '05:05' HOUR_MINUTE ) ) ) IS NOT FALSE ) ) OR NOT ( col_timestamp_6 IN (DATE_SUB( DATE_SUB( CONCAT( CAST( 2015 + 3 + 2 AS CHAR(20) ) , '-06-19 01:00:00.000003') , INTERVAL '05:05.000005' MINUTE_MICROSECOND ) , INTERVAL '05:05:05.000005' HOUR_MICROSECOND ) , DATE_ADD( CONCAT( CAST( 2015 + 5 + 3 AS CHAR(20) ) , '-06-19 01:00:00.000003') , INTERVAL '01 05:05:05' DAY_SECOND ) , CONCAT( CAST( 2015 + 5 AS CHAR(20) ) , '-06-15 01:00:00.000003')) ) ORDER BY 1 ASC , pk  LIMIT 3;
```

```
CREATE TABLE `table1_hash_to_days_pk_to_days_pk_subpart_by_range_columns_pk_su` (
  `col_timestamp_6` timestamp(6) NULL DEFAULT NULL,
  `col_datetime` datetime DEFAULT NULL,
  `col_bigint_unsigned` bigint(20) unsigned DEFAULT NULL,
  `col_decimal_20_0` decimal(20,0) DEFAULT NULL,
  `pk` datetime NOT NULL,
  `col_decimal_20_0_unsigned` decimal(20,0) unsigned DEFAULT NULL,
  `col_char_20` char(20) DEFAULT NULL,
  `col_bigint` bigint(20) DEFAULT NULL,
  `col_decimal_20_0_signed` decimal(20,0) DEFAULT NULL,
  `col_bigint_signed` bigint(20) DEFAULT NULL,
  PRIMARY KEY (`pk`)
) DEFAULT CHARSET = utf8mb4 ROW_FORMAT = DYNAMIC COMPRESSION = 'zstd_1.3.8' REPLICA_NUM = 2 BLOCK_SIZE = 16384 USE_BLOOM_FILTER = FALSE TABLET_SIZE = 134217728 PCTFREE = 0
 partition by hash(to_days(pk) * to_days(pk)) subpartition by range columns(pk) subpartition template (
subpartition `p0` values less than ('2010-01-01 00:00:00'),
subpartition `p1` values less than ('2012-01-01 00:00:00'),
subpartition `p2` values less than ('2014-01-01 00:00:00'),
subpartition `p3` values less than (MAXVALUE))
(partition `p0`,
partition `p1`,
partition `p2`,
partition `p3`,
partition `p4`,
partition `p5`,
partition `p6`,
partition `p7`,
partition `p8`,
partition `p9`,
partition `p10`,
partition `p11`,
partition `p12`,
partition `p13`,
partition `p14`,
partition `p15`,
partition `p16`,
partition `p17`,
partition `p18`,
partition `p19`,
partition `p20`,
partition `p21`,
partition `p22`,
partition `p23`,
partition `p24`,
partition `p25`,
partition `p26`,
partition `p27`,
partition `p28`,
partition `p29`,
partition `p30`,
partition `p31`,
partition `p32`,
partition `p33`,
partition `p34`,
partition `p35`,
partition `p36`,
partition `p37`,
partition `p38`,
partition `p39`,
partition `p40`,
partition `p41`,
partition `p42`,
partition `p43`,
partition `p44`,
partition `p45`,
partition `p46`,
partition `p47`,
partition `p48`,
partition `p49`)
```

**Comment by HSunboy** [comment]
Fixed In Commit [09afd36b05834f891e0ddbee160d72df33d38768](https://github.com/oceanbase/odc-client/commit/09afd36b05834f891e0ddbee160d72df33d38768)
            ------
            xiaokang
            Mon Aug 12 15:40:18 2024 +0800

**Comment by Huangxiao-mas** [comment]
pass
